lib/usd/ui/layerEditor/layerTreeModel.cpp

```diff
@@ -481,8 +482,11 @@ void LayerTreeModel::saveStage(QWidget* in_parent)
     auto saveAllLayers = [this]() {
         const auto layers = getAllNeedsSavingLayers();
         for (auto layer : layers) {
-            if (!layer->isAnonymous())
-                layer->saveEditsNoPrompt();
+            if (!layer->isSystemLocked()) {
```

Comment:
A System-locked layer cannot be saved by the extension that PermissionToSave = False.